### PR TITLE
Homework 2,3

### PR DESCRIPTION
**Pull request by Viktoriia-Zinchenko**

### CODE REVIEW

File: src/main/java/com/java_essential/ZinchenkoVika/Classwork/lesson2/Student.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package com.java_essential.ZinchenkoVika.Classwork.lesson2;
+
+ class Student {
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/ZinchenkoVika/Classwork/lesson2/Student.java:4]: Clear this empty space.

File: src/main/java/com/java_essential/ZinchenkoVika/Classwork/lesson2/Student.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        public void setLastName(String lastName) {
+        this.lastName = lastName;
+        }
+        public int getAge() {
+        return age;
+        }
+        public void setAge(int age) {
+        this.age = age;
+        }
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/ZinchenkoVika/Classwork/lesson2/Student.java:65]: CTRL + ALT + L
Add separator between methods.

File: src/main/java/com/java_essential/ZinchenkoVika/Homework/lesson2/Car.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        this.year = year;
+        this.speed = speed;
+        this.weight = weight;
+    }
+    public Car(int year, double speed, int weight, String color) {
+        this.year = year;
+        this.speed = speed;
+        this.weight = weight;
+        this.color = color;
+    }
```
Comment (ochedenis) [inline src/main/java/com/java_essential/ZinchenkoVika/Homework/lesson2/Car.java:31]: Add separator between constructors.

**Review by ochedenis — CHANGES_REQUESTED** [review]

File: src/main/java/Homework/lesson3/Vehicle/Main.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        car.setYear(2000);
+        System.out.println("Car: price - " + car.getPrice() + ", speed - " + car.getSpeed() + ", year - " + car.getYear());
+        // For plane:
+        Plane plane = new Plane();
+        plane.setPrice(1540000);
+        plane.setSpeed(1245);
+        plane.setYear(1999);
+        plane.setHeight(1500);
+        plane.setPassengersPlane(6);
+        System.out.println("Plane: price - " + plane.getPrice() + ", speed - " + plane.getSpeed() + ", year - " + plane.getYear() + ", height - " + plane.getHeight() + ", passengers - " + plane.getPassengersPlane());
```
Comment (yuriy-tsarenko) [inline src/main/java/Homework/lesson3/Vehicle/Main.java:18]: separate lines that are too long

File: src/main/java/Homework/lesson3/Vehicle/Main.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        // For ship:
+        Ship ship = new Ship();
+        ship.setPrice(284650000);
+        ship.setSpeed(90);
+        ship.setYear(2015);
+        ship.setPassengersShip(1500);
+        ship.setPortOfRegistration("Odessa");
+        System.out.println("Ship: price - " + ship.getPrice() + ", speed - " + ship.getSpeed() + ", year - " + ship.getYear() + ", passengers - " + ship.getPassengersShip() + ", port of registration - " + ship.getPortOfRegistration());
```
Comment (yuriy-tsarenko) [inline src/main/java/Homework/lesson3/Vehicle/Main.java:26]: separate lines that are too long

**Review by yuriy-tsarenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**